src/frontend/index.tsx

```diff
@@ -28,9 +28,11 @@ if (process.env.NODE_ENV !== 'production') {
     axe(React, ReactDOM, 1000);
 }
 
-ReactDOM.render(
+const container = document.getElementById('app');
+const root = ReactDOM.createRoot(container!); // eslint-disable-line @typescript-eslint/no-non-null-assertion
```

Comment:
Dette er hovedendringen ved oppgradering til `react-dom` 18
https://reactjs.org/blog/2022/03/08/react-18-upgrade-guide.html#updates-to-client-rendering-apis